mlflow/fastai.py

```diff
@@ -77,6 +79,25 @@ def save_model(fastai_learner, path, conda_env=None, mlflow_model=Model(), **kwa
                             ]
                         }
     :param mlflow_model: MLflow model config this flavor is being added to.
+
+    :param signature: (Experimental) :py:class:`ModelSignature <mlflow.models.ModelSignature>`
```

Comment:
BTW, do we also plan to document these params for `log_model` in a follow-up (IMO we should)?

Reply:
what do you mean? You mean to update the models.rst docs?

Reply:
Basically, we probably want to update the docs for [`mlflow.fastai.log_model`](https://github.com/mlflow/mlflow/blob/66a54d5aa5829abe06b8020a36e328ccb031397f/mlflow/fastai.py#L109) & analogous methods in other flavors to mention that `signature` and `input_example` are valid arguments (& actually in the fastai case, that they won't be passed to `fastai.Learner.export` [as the docs currently state](https://github.com/mlflow/mlflow/blob/66a54d5aa5829abe06b8020a36e328ccb031397f/mlflow/fastai.py#L135))

Reply:
Oops, sorry I somehow missed that this PR also updates the log_model docstrings - nvm!